lib/data/entities/conversation.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'message.dart';
+
+class ConversationSnippet {
+  final String id;
+  final String conversationID;
+  final String lastMessage;
+  final String name;
+  final String image;
+  final MessageType type;
+  final int unseenCount;
+  final Timestamp timestamp;
+
+  ConversationSnippet(
+      {this.conversationID,
+        this.id,
+        this.lastMessage,
+        this.unseenCount,
+        this.timestamp,
+        this.name,
+        this.image,
+        this.type});
```

Comment:
```suggestion
        this.type,});
```
should be add comma at the end of the last param and format code